modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -24,10 +24,14 @@ When you hover over the `i` to the right of each rule, the rule ID range for tha
 +
 image::waf-rule-set.png[]
 +
-. In *Advanced performance options*, you can select *Disable body scanner*. If you do not check this option, the request body is scanned.
+. In *Advanced performance options*, select from the following options: +
+* *Disable body scanner*. If you do not check this option, the request body is scanned.
 +
 [NOTE]
-The *Disable body scanner* option must be checked if any valid responses will exceed 512 KB. If request body checking is enabled, the HTTP Limits policy “Maximum Message Size” should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting the resources.
+If a body is larger than 1 MB, the scan is skipped. If request body checking is enabled, the HTTP Limits policy *Maximum Message Size* should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting resources.
```

Comment:
You don't have to select either one. If Disable body scanner is selected, the body isn't scanned at all. Nevertheless it's confusing because why are we talking about this in the Disable body scanner point? If you disable the body scanner, it doesn't happen at all--so yes, regardless, the scan won't happen.